Clients/webfrontauth/ts/src/AuthService.ts

```diff
@@ -448,21 +447,29 @@ export class AuthService<T extends IUserInfo = IUserInfo> {
                 }
             }
 
-            const eOnClick = popup.document.getElementById('submit-button'); 
+            const eOnClick = popup.document.getElementById('submit-button');
             if( eOnClick == null ) throw new Error( "Unable to find required 'submit-button' element." );
             eOnClick.onclick = (async () => await onClick());
-        } 
+        }
         else {
             const url = `${this._configuration.webFrontAuthEndPoint}.webfront/c/startLogin`;
             userData = { ...userData, callerOrigin: document.location.origin, rememberMe: rememberMe };
-            const queryString = userData 
+            const queryString = userData
                                 ? Object.keys(userData).map((key) => encodeURIComponent(key) + '=' + encodeURIComponent(userData![key] as string)).join('&')
                                 : '';
             const finalUrl = url + '?scheme=' + scheme + ((queryString !== '') ? '&' + queryString : '');
             window.open(finalUrl, this.popupDescriptor.popupTitle, this.popupDescriptor.features);
         }
     }
 
+    /**
+     * Checking that url is the same as configuration to authorize the addition of the token.
```

Comment:
Checks whether calling the provided url requires the header bearer token to be added or not.
Currently, only urls from the authentication endpoint should add the authentication token (the url must exactly starts with the authentication backend address)  but this may be extended to support other secure endpoints if needed.